Dockerfile

```diff
@@ -19,9 +19,13 @@ FROM tomcat:9
 
 RUN apt-get update && apt-get upgrade -y
 
-RUN wget -O dd-java-agent.jar https://dtdg.co/latest-java-tracer
+RUN wget -O dd-java-agent.jar https://dtdg.co/latest-java-tracer && \
+    wget -O /usr/local/tomcat/bin/jul-ecs-formatter-1.6.0.jar https://repo1.maven.org/maven2/co/elastic/logging/jul-ecs-formatter/1.6.0/jul-ecs-formatter-1.6.0.jar && \
+    wget -O /usr/local/tomcat/bin/ecs-logging-core-1.6.0.jar https://repo1.maven.org/maven2/co/elastic/logging/ecs-logging-core/1.6.0/ecs-logging-core-1.6.0.jar
```

Comment:
Try to avoid adding new jars directly. Add the dependency within pom.xml instead: https://mvnrepository.com/artifact/co.elastic.logging/ecs-logging-core/1.6.0

Reply:
Those have to be accessible on the path somewhere so that the JVM logs can be formatted. Note - this is only necessary for the tomcat/catalina log output formatting and not the application so technically not a part of the project (e.g. ef you were to migrate to spring boot this would become redundant). 

I can add those via maven - do you know where those jar then end up as I will have to expose them to the JVM by putting them on the path somewhere (see setenv.sh)

Reply:
The jars will be part of a WAR package that will be running from Tomcat.
With the current approach your logger will work only within a Docker container with unavailability to test it locally (unless you provide the missing jars manually).

Reply:
Regarding the Spring-boot update, I may certainly assist you with that. Feel free to get back to me by email.

Reply:
Maybe I was unclear here - These jar files are only for tomcat log output, and not application logs.